README.md

```diff
@@ -1 +1,57 @@
-# StoopidProject
```

Comment:
Why is this red?

Reply:
Red shows up as a line being deleted. I modified the file and added more lines so Git thinks the line got changed.

Reply:
gotcha, thanks!